amivapi/blacklist/model.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+#
+# license: AGPLv3, see LICENSE for details. In addition we strongly encourage
+#          you to buy us beer if we meet and you like the software.
+"""Blacklist settings.
+
+Contains model for blacklist.
+"""
+from amivapi.blacklist.authorization import BlacklistAuth
+
+
+blacklist_description = ("""
+People normally get blacklisted if they don't appear to an event they signed up
+for, but other cases could be possible (Bad behaviour, etc). Once on the
+blacklist, they shouldn't be able to sign up for any event until they pay
+for that event or do something else the board decides (e.g. help at an event).
+""")
```

Comment:
The description is a bit short.

In particular, it does not cover:

1. That a user can be on the blacklist for multiple offences at once.
2. That the blacklist entry is defined by a start and end date
3. That users can only see their (own) blacklist entries, not others, and that only admins can set the dates.

Have a look at the other resource descriptions if you need inspiration :)